### PR TITLE
added stopwords language argument for do_tokenize and do_tokenize_icu

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

Added stopwords language argument to do_tokenize and do_tokenize_icu

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/string_operation.R [hunk 1]
```diff
@@ -27,6 +27,9 @@
 #' @return Logical vector if the token is in stop words or not.
 #' @export
 is_stopword <- function(token, lang = "english", include = c(), exclude = c(), ...){
+  if(lang == "japanese") { # for Japanese, assume the token is stopword if it's one letter
```
Comment (hideaki) [inline R/string_operation.R]:
```suggestion
  if(lang == "japanese") { # for Japanese, assume the token is stopword if it's a short consecutive hiragana-only word.
```

File: R/string_operation.R [hunk 2]
```diff
@@ -91,7 +95,8 @@ get_stopwords <- function(lang = "english", include = c(), exclude = c(), is_twi
   if(is_twitter) {
     stopwords <- append(stopwords, exploratory_stopwords)
   }
-  ret <- c(stopwords[!stopwords %in% exclude], include)
+  # tidystopwords required language name with Titlt Case so make sure it's title case.
```
Comment (hideaki) [inline R/string_operation.R]:
```suggestion
  # tidystopwords required language name with Title Case so make sure it's title case.
```

**Comment by hideaki** [comment]
Errors are  from lack of tidystopwords.
```
⠏ |   0       | test string operation functionsFailed with error:  'there is no package called 'tidystopwords''
Failed with error:  'there is no package called 'tidystopwords''
Failed with error:  'there is no package called 'tidystopwords''
```

File: R/string_operation.R [hunk 3]
```diff
@@ -272,6 +296,12 @@ do_tokenize <- function(df, input, token = "words", keep_cols = FALSE,  drop = T
   # the following does not work with error that says := is unknown. do it base-R way.
   # ret <- ret %>% dplyr::rename(!!rlang::sym(orig_input_col) := !!rlang::sym(input_col))
   colnames(ret)[colnames(ret) == input_col] <- orig_input_col
+  # if stopwords_lang is provided, remove the stopwords for the language.
+  if(!is.null(stopwords_lang)) {
+    stop_words <- exploratory::get_stopwords(lang = stopwords_lang)
+    ret <- ret %>% dplyr::filter(!!as.name(output_col) %nin% stop_words)
```
Comment (hideaki) [inline R/string_operation.R]:
```suggestion
    ret <- ret %>% dplyr::filter(!!sym(output_col) %nin% stop_words)
```

File: R/string_operation.R [hunk 4]
```diff
@@ -272,6 +296,12 @@ do_tokenize <- function(df, input, token = "words", keep_cols = FALSE,  drop = T
   # the following does not work with error that says := is unknown. do it base-R way.
   # ret <- ret %>% dplyr::rename(!!rlang::sym(orig_input_col) := !!rlang::sym(input_col))
   colnames(ret)[colnames(ret) == input_col] <- orig_input_col
+  # if stopwords_lang is provided, remove the stopwords for the language.
+  if(!is.null(stopwords_lang)) {
+    stop_words <- exploratory::get_stopwords(lang = stopwords_lang)
+    ret <- ret %>% dplyr::filter(!!rlang::sym(output_col) %nin% stop_words)
```
Comment (hideaki) [inline R/string_operation.R]: Can is_stopword be reused here?

Comment (hidekoji) [reply R/string_operation.R]: Changed to use is_stopword as suggested

**Pull request merged**